.nvmrc

```diff
@@ -0,0 +1 @@
+16.13.2
```

Comment:
Call `nvm use` next time you check out the project. If you don't have NVM install it here https://github.com/nvm-sh/nvm